src/refiners/fluxion/utils.py

```diff
@@ -113,6 +114,12 @@ def default_sigma(kernel_size: int) -> float:
     return tensor
 
 
+def images_to_tensor(
+    images: List[Image.Image], device: Device | str | None = None, dtype: DType | None = None
+) -> Tensor:
```

Comment:
Since we target 3.10+, `list[Image.Image]` is enough (and you should drop the `from typing import List` of course):

```python
def images_to_tensor(
    images: list[Image.Image], device: Device | str | None = None, dtype: DType | None = None
) -> Tensor:
```